### PR TITLE
fix logging for workers

### PR DESCRIPTION
**Pull request by jlvallelonga**
workers weren't logging correctly so this fixes them

### CODE REVIEW

**Comment by jlvallelonga** [comment]
solidqueue is still logging heartbeat updates every minute so I'll continue working on a fix for that, but wanted to push this as a draft first

File: config/initializers/solid_queue_extension.rb
Comment (jlvallelonga) [inline config/initializers/solid_queue_extension.rb:1]: I'll try to get a PR in for solidqueue to make this an official option

**Review by krschacht — APPROVED** [review]
👍 looks good, if you’re done with this one feel free to merge in. I couldn’t tell if there was something else you wanted to add to it.

File: config/initializers/solid_queue_extension.rb
Comment (jlvallelonga) [reply config/initializers/solid_queue_extension.rb:1]: https://github.com/rails/solid_queue/pull/389

**Pull request merged**